release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1082,40 +1103,35 @@ In the following tables, features are marked with the following statuses:
 |====
 
 [discrete]
-=== Operator Technology Preview features
+=== Operator lifecycle and development Technology Preview features
 
-.Operator Technology Preview tracker
+.Operator lifecycle and development Technology Preview tracker
 [cols="4,1,1,1",options="header"]
 |====
 |Feature |4.12 |4.13 |4.14
 
-|Hybrid Helm Operator
-|Technology Preview
-|Technology Preview
+|Operator Lifecycle Manager (OLM) v1
+|Not Available
+|Not Available
 |Technology Preview
 
-|Java-based Operator
+|RukPak
 |Technology Preview
 |Technology Preview
 |Technology Preview
 
-|Multi-cluster Engine Operator
```

Comment:
I am not sure why "Multi-cluster Engine Operator" is here in the "Operator lifecycle and development Technology Preview tracker", so I moved it down to the "Architecture Technology Preview features" section since it seemed like it is related to hosted control planes, but I will follow up with the related tech writer to confirm that before merge.